src/target.py

```diff
@@ -614,7 +614,7 @@ def stage(self, mvals, AGrad=False, AHess=False, customdir=None, firstIteration=
         cwd = os.getcwd()
 
         absgetdir = os.path.join(self.root,self.tempdir)
-        if Counter() is not None:
+        if use_iterdir and Counter() is not None:
```

Comment:
@j-wags This was one of the changes; the `stage()` function automatically goes into an "iteration" folder but that is not where the `get()` function looks for the data in the test, so I had to add an extra keyword argument.